core/api/src/main/java/com/alipay/sofa/rpc/context/AsyncRuntime.java

```diff
@@ -100,4 +100,13 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor executor) {
         }
         return asyncThreadPool;
     }
+
+    /**
+     * 设置callback用的线程池
+     *
+     * @param asyncThreadPool callback用的线程池
+     */
+    public static void setAsyncThreadPool(ThreadPoolExecutor asyncThreadPool) {
+        AsyncRuntime.asyncThreadPool = asyncThreadPool;
```

Comment:
We need to consider a more reasonable configuration for supporting asynchronous thread pool setups. The changes here do not guarantee thread safety and do not ensure that no other logic will have access to the thread pool created by the original logic before the set method is called. I hope to have a modified solution to discuss before putting it into development.

Reply:
> We need to consider a more reasonable configuration for supporting asynchronous thread pool setups. The changes here do not guarantee thread safety and do not ensure that no other logic will have access to the thread pool created by the original logic before the set method is called. I hope to have a modified solution to discuss before putting it into development.

sorry, I will follow the steps.